noxfiles/ci_nox.py

```diff
@@ -35,8 +35,8 @@ def static_checks(session: nox.Session) -> None:
 @nox.parametrize(
     "mode",
     [
-        nox.param("fix", id="fix"),
```

Comment:
this change in order turns out to not technically be needed but i figure i'd keep it here to make it consistent with the order in `fidesplus`